src/undomanager.js

```diff
@@ -1,12 +1,14 @@
 define([
     'jquery',
     'underscore',
+    'vellum/util',
     'tpl!vellum/templates/undo_alert',
 ], function(
     $,
     _,
-    UNDO_ALERT
-) {
+    util,
+    UNDO_ALERT,
+undefined) {
```

Comment:
I understand and agree with your appreciation for trailing commas, but I think this is aesthetically...not very pleasant. I'm -1 on this as a convention. Also, I think in modern browsers it won't, but might this possibly define `undefined` if too many arguments are passed to the function?